src/Components/Notes/NoteCard.test.jsx

```diff
@@ -58,4 +63,22 @@ describe('NoteCard', () => {
     const showCamera = rendered.getByTitle('Show the camera view')
     expect(showCamera).toBeInTheDocument()
   })
+
+  it('Delete button is present', () => {
+    const id = 123
+    const index = 123
+    const username = 'testing'
+    mockedUseAuth0.mockReturnValue(mockedUserLoggedIn)
+
+    const {getByTitle} = render(
+        <ShareMock>
+          <NoteCard
+            id={id}
+            index={index}
+            username={username}
+            synched={true}
+          />
+        </ShareMock>)
+    expect(getByTitle('Delete')).toBeInTheDocument()
+  })
 })
```

Comment:
All looking good!  As discussed, pls add a unit test for clicking the delete and expecting the github mock delete is called.  Then i think it's ready for merge

Reply:
Add the test for the delete.

Reply:
Rad!